uint16 cmd_L
uint16 cmd_R
bool isControl
